daemon/logger/fluentd/fluentd.go

```diff
@@ -24,10 +24,12 @@ type fluentd struct {
 }
 
 const (
-	name             = "fluentd"
-	defaultHostName  = "localhost"
-	defaultPort      = 24224
-	defaultTagPrefix = "docker"
+	name                             = "fluentd"
+	defaultHostName                  = "localhost"
+	defaultPort                      = 24224
+	defaultTagPrefix                 = "docker"
+	defaultIgnoreConnectErrorOnStart = false           // So that we do not break existing behaviour
+	defaultBufferLimit               = 8 * 1024 * 1024 // 8M buffer by default
```

Comment:
The fluent driver itself uses 8M, that's why I chose it also.
https://github.com/fluent/fluent-logger-golang/blob/master/fluent/fluent.go#L21

For me it seems appropriate to use the same default as the fluent driver but I can change it to 1M if that is still seen as better option?